src/base/static/components/story-sidebar/story-chapter.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+
+import "./story-chapter.scss";
+
+const StoryChapter = props => {
+  return (
+    <a
+      className={classNames("story-chapter", {
+        "story-chapter--selected": props.isSelected,
+      })}
+      href={"/" + props.placeUrl}
+      rel="internal"
+    >
```

Comment:
nit: consider making this `a` tag a `Link` component, which will live in our `atoms` dir.